src/commands/mod.rs

```diff
@@ -1387,7 +1387,7 @@ fn log_template(settings: &UserSettings) -> String {
             if(branches, " " branches)
             if(tags, " " tags)
             if(working_copies, " " working_copies)
-            if(is_git_head, label("git_head", " HEAD@git"))
+            if(git_head, label("git_head", " " git_head))
```

Comment:
nit: `label("git_head")` is redundant because the keyword is labeled by default?

Reply:
Good catch. Looks like I also incorrectly labeled the space (even before this PR).